src/util/unicode.h

```diff
@@ -20,7 +20,6 @@ std::string narrow(const std::wstring &s);
 std::wstring widen(const std::string &s);
 
 std::string utf32_to_utf8(const std::basic_string<unsigned int> &s);
```

Comment:
I may well be - there is a function that has been renamed and I cannot see a corresponding change in the header file.